src/TestResults/TestResultProvider.ts

```diff
@@ -92,6 +92,16 @@ export class TestResultProvider implements JestExtSessionAware {
     this.testSuites = {};
   }
 
+  isTestFile(fileName: string): 'yes' | 'no' | 'unknown' {
```

Comment:
I'm not completely happy with this return type. But compared to `boolean | undefined` or an enum, this seems to be the best solution.

Reply:
yeah, I have also considered `boolean | "unknown"` but then the `if(isTestFile())` will be true for both `true or "unknown"`, so decided to just make it more explicit with a string literal type...

thanks for the quick review! 👍